packages/cli/src/commands/setup/deploy/__tests__/netlify.test.js

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+import path from 'path'
+
+const FIXTURE_PATH = path.resolve(
+  __dirname,
+  './__fixtures__/example-netlify-deploy-project'
+)
+
+beforeAll(() => {
+  process.env.RWJS_CWD = FIXTURE_PATH
+})
+
+afterAll(() => {
+  delete process.env.RWJS_CWD
+})
+
+describe('netlify', () => {
+  it('should call the handler without error', async () => {
+    const netlify = require('../providers/netlify')
+    expect(async () => await netlify.handler({ force: true })).not.toThrow()
```

Comment:
This will result in a slow test I think (the problems in our generators) - is it worth mocking out more of the methods used inside the handler?

Reply:
More mocks have been added